Blockzilla/TelemetryIntegration.swift

```diff
@@ -22,6 +22,8 @@ class TelemetryEventMethod {
     public static let open = "open"
     public static let openAppStore = "open_app_store"
     public static let share = "share"
+    public static let navigateBack = "navigate_back"
```

Comment:
I wasn't sure about the telemetry methods names, but you suggestion looks good. I'll fix.